main/streams/userspace.c

```diff
@@ -692,6 +697,12 @@ static size_t php_userstreamop_read(php_stream *stream, char *buf, size_t count
 			1, args,
 			0, NULL TSRMLS_CC);
 
+	zval_ptr_dtor(&zcount);
+
+	if (EG(exception)) {
+		return -1;
```

Comment:
@smalyshev there is also some other issues, I have told @jpauli via IRC.
(like exception check in write should move above to zvaL-ptr_dtor(&zbufptr)

I think he will make some improvements.. so please hold from merging this for now.

thanks